test/view_component/view_component_test.rb

```diff
@@ -386,6 +386,61 @@ def test_backtrace_returns_correct_file_and_line_number
     assert_match %r[app/components/exception_in_template_component\.html\.erb:2], error.backtrace[0]
   end
 
+  # Rendering collections
+
+  def test_render_collection
+    @products = [OpenStruct.new(name: "Radio clock"), OpenStruct.new(name: "Mints")]
```

Comment:
Its a nit to pik but do we need instance variables here? can't we just use regular variables given they're internal to the test method.

Reply:
Good catch! Fixed.